molecule/tests/test_default.py

```diff
@@ -45,13 +43,22 @@ def test_files_and_dirs(host, path):
 
 
 @pytest.mark.parametrize(
-    "service,isEnabled", [("clamav-daemon", False), ("clamav-freshclam", True)]
+    "service,is_enabled", [("clamav-daemon", False), ("clamav-freshclam", True)]
 )
-def test_services(host, service, isEnabled):
+def test_services_debian(host, service, is_enabled):
     """Test that the expected services were enabled or disabled as intended."""
-    if (
-        host.system_info.distribution == "debian"
-        or host.system_info.distribution == "ubuntu"
-    ):
+    if host.system_info.distribution in ["debian", "ubuntu"]:
```

Comment:
Should this be in here?

```suggestion
    if host.system_info.distribution in ["debian", "kali", "ubuntu"]:
```